source/index.html.md

```diff
@@ -73,7 +73,7 @@ The other content you see in the source folder is the site for this tutorial. We
     ...
 ```
 
-> Go to your GitHub account, select Repositories tab and select smart-on-fhir-tutorial repo. Select Branch button and switch to gh-pages branch. Directly edit `/example-smart-app/index.html` by clicking on the pencil icon.  Once done with the change, commit directly to gh-pages branch.
+> Go to your GitHub account, select Repositories tab and select smart-on-fhir-tutorial repo. Select Branch button and switch to gh-pages branch if it is not already selected. Directly edit `/example-smart-app/index.html` by clicking on the pencil icon.  Once done with the change, commit directly to gh-pages branch.
```

Comment:
How about adding the following : 
```Once done with the change, commit directly to gh-pages branch this will ensure your changes are auto deployed by github. ```

Reply:
OK.